ui/ui.gradle.kts

```diff
@@ -349,7 +338,8 @@ fun JpackageExec.configureForCurrentOs() {
         OperatingSystem.LINUX -> {
             val installerFileDir = installerFilesBaseDir.resolve("linux")
             resourceDir.set(installerFileDir)
-            icon.set(installerFileDir.resolve("GRIP.png"))
+            // Skip icon on linux, causing build to break
+            //icon.set(installerFileDir.resolve("GRIP.png"))
```

Comment:
Think you can figure it out/fix it?

Reply:
I don't have a linux system, so my only testing is on Azure, which is basically impossible to fix. Would need someone else to test it and figure out whats happening.

Reply:
I think its a bug in the j14 jpackager. 

```
Using custom package resource [menu icon]  (loaded from GRIP.png).
java.nio.file.FileAlreadyExistsException: /tmp/jdk.jpackage13600112042171272034/images/opt/grip/lib/GRIP.png
	at java.base/sun.nio.fs.UnixException.translateToIOException(UnixException.java:94)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:111)
	at java.base/sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:116)
	at java.base/sun.nio.fs.UnixFileSystemProvider.newByteChannel(UnixFileSystemProvider.java:219)
	at java.base/java.nio.file.spi.FileSystemProvider.newOutputStream(FileSystemProvider.java:478)
	at java.base/java.nio.file.Files.newOutputStream(Files.java:223)
	at java.base/java.nio.file.Files.copy(Files.java:3143)
	at jdk.jpackage/jdk.jpackage.internal.AbstractBundler.fetchResource(AbstractBundler.java:94)
	at jdk.jpackage/jdk.jpackage.internal.LinuxPackageBundler$DesktopIntegration.prepareSrcIconFile(LinuxPackageBundler.java:771)
	at jdk.jpackage/jdk.jpackage.internal.LinuxPackageBundler$DesktopIntegration.create(LinuxPackageBundler.java:432)
	at jdk.jpackage/jdk.jpackage.internal.LinuxPackageBundler.execute(LinuxPackageBundler.java:177)
	at jdk.jpackage/jdk.jpackage.internal.Arguments.generateBundle(Arguments.java:627)
	at jdk.jpackage/jdk.jpackage.internal.Arguments.processArguments(Arguments.java:513)
	at jdk.jpackage/jdk.jpackage.main.Main.execute(Main.java:98)
	at jdk.jpackage/jdk.jpackage.main.Main.main(Main.java:51)
jdk.jpackage.internal.PackagerException: java.nio.file.FileAlreadyExistsException: /tmp/jdk.jpackage13600112042171272034/images/opt/grip/lib/GRIP.png
```

Its failing to copy the icon file to the temp folder, but I can't find anywhere where its already being copied.